### ISSUE TITLE
Version 2.11.24 breaks CLI Authenticated Deploys

### ISSUE THREAD

**Issue by arjansingh** [netlify/cli#356]
**- Do you want to request a *feature* or report a *bug*?**

*BUG*

**- What is the current behavior?**

If you run `netlify deploy --auth=<token>` the auth token is not properly parsed and you get a `401` error from your site.

This behavior previously worked in `netlify-cli@2.11.23`.

**- If the current behavior is a bug, please provide the steps to reproduce.**

1. Install `netlify-cli@2.11.24`
2. Try to run a `netlify deploy` command where you specify your authentication token via the `--auth` parameter.
3. Observe the returned message:
```
 ›   Error: invalid json response body at https://api.netlify.com/api/v1/sites/13343-foo-bar-baz reason: Unexpected
 ›   end of JSON input
```

**- What is the expected behavior?**

The deploy should succeed.

**- Please mention your node.js, and operating system version.**
Node 10.16
OS: Mac OSX 10.14 and Debian GNU/Linux 9

**Comment by DavidWells** [comment]
This should be fixed with netlify-cli@2.11.25 can you update and verify?

**Comment by DavidWells** [comment]
Looks like there is an [NPM publishing delay](https://npm.community/t/no-matching-version-found-but-package-version-is-published/4677/2) 😢

---
A new version of the package netlify-cli (2.11.25) was published at 2019-07-17T07:49:25.668Z from
67.188.35.102. The shasum of this package was fc59ab95cd2c42cffa2002dc05b8afae779fb66d.
---

Version 2.11.25 is live https://www.npmjs.com/package/netlify-cli/v/2.11.25 but I can't install it yet. This should resolve itself

---

A temporary workaround is setting `NETLIFY_AUTH_TOKEN` env var

```
export NETLIFY_AUTH_TOKEN=your-token 
netlify deploy -p
```


Sorry for the troubles.

**Comment by arjansingh** [comment]
Confirmed that this works. Thanks.
